egs/swbd/s5c/local/nnet3/run_cnn.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# Copyright 2015  Johns Hopkins University (Author: Daniel Povey).
+#           2015  Vijayaditya Peddinti
+#           2016  Tom Ko
+# Apache 2.0.
+
+# this is the CNN+TDNN system, built in nnet3;
+# this system differs from the standard TDNN system
+# in replacing the LDA layer with CNN 
+# and using fbank features to the CNN input
```

Comment:
Also there is a change in how ivectors are used. Better name the script run_cnn_tdnn.sh